core/src/main/scala/com/yahoo/maha/core/DerivedExpression.scala

```diff
@@ -562,6 +562,12 @@ object HiveExpression {
     def asString : String = s"from_unixtime(unix_timestamp(${s.asString}, '$fmt'), 'EEEE')"
   }
 
+  case class TIME_FORMAT_WITH_TIMEZONE(s: HiveExp, fmt: String, timezone: String = "UTC") extends BaseHiveExpression {
+    val hasRollupExpression = s.hasRollupExpression
+    val hasNumericOperation = s.hasNumericOperation
+    def asString : String = s"from_unixtime(unix_timestamp(from_utc_timestamp(from_unixtime(unix_timestamp(${s.asString}, 'yyyyMMddHH'), 'yyyy-MM-dd HH:mm:ss'), '$timezone')), '$fmt')"
```

Comment:
what is the use case for 3 level nested timestamp conversion? is there any option to honor another timezones?